regparser/builder.py

```diff
@@ -237,7 +238,7 @@ def tree_and_builder(filename, title, checkpoint_path=None, doc_number=None):
     else:
         raise ValueError("Building from text input is no longer supported")
 
-    for preprocessor in all_preprocs.ALL:
+    for preprocessor in class_paths_to_classes(settings.PREPROCESSORS):
```

Comment:
I agree. I don't think this is a particularly good time to try that refactor, though.